AudioSinkExample/ViewController.swift

```diff
@@ -0,0 +1,705 @@
+//
+//  ViewController.swift
+//  AudioSinkExample
+//
+//  Copyright © 2017 Twilio Inc. All rights reserved.
+//
+
+import AVFoundation
+import TwilioVideo
+import UIKit
+
+class ViewController: UIViewController {
+
+    // MARK: View Controller Members
+
+    // Configure access token manually for testing, if desired! Create one manually in the console
+    // at https://www.twilio.com/console/video/runtime/testing-tools
+    var accessToken = "TWILIO_ACCESS_TOKEN"
+
+    // Configure remote URL to fetch token from
+    var tokenUrl = "http://localhost:8000/token.php"
+
+    // Automatically record audio for all `TVIAudioTrack`s published in a Room.
+    var recordAudio = true
+
+    // Video SDK components
+    var room: TVIRoom?
+    var camera: TVICameraCapturer?
+    var localAudioTrack: TVILocalAudioTrack!
```

Comment:
Why are these suffixed with `!` and not `?` ?

Reply:
They should be `?`. I believe I coped these vars from other sample code. 😰